src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -166,6 +176,29 @@ export const Deposit: React.FC<DepositProps> = ({
     selectPortfolioCryptoBalanceBaseUnitByFilter(state, balanceFilter),
   )
 
+  const liquidityLockupTime = useQuery({
+    ...reactQueries.thornode.mimir(),
+    staleTime: thorchainBlockTimeMs,
+    select: mimirData => {
+      const liquidityLockupBlocks = mimirData.LIQUIDITYLOCKUPBLOCKS as number | undefined
+      return Number(bnOrZero(liquidityLockupBlocks).times(THORCHAIN_BLOCK_TIME_SECONDS).toFixed(0))
+    },
```

Comment:
selector is duplicated across 4 modules, can we consolidate either the selector as `selectLiquidityLockuptime`, or the query as `useLiquidityLockupTimeQuery()` ?

Reply:
https://github.com/shapeshift/web/pull/7762/commits/5fd21c91ce3dec3343ff76d63d5a6f9dbee6a12e